src/twisted/newsfragments/11997.bugfix

```diff
@@ -0,0 +1 @@
+fix for Twisted web client support of trailer Server-Timing
```

Comment:
We are using this text as part of our public release notes.

Can you please write it as a full sentence and add a few info about how was  the previous behaviour and how it behaves how?